# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1428611816 14400
#      Thu Apr 09 16:36:56 2015 -0400
# Node ID df3244834a5025d63a15c20991ff2f14ad265158
# Parent  438485f6a47951319d410fea89c684ec6944805c
fix tests now that assert ([]) fails

* uimenu.m: Fix test now that FLTK menu bar doesn't have a Help menu.
* isbanded.m: Fix test for [] result to actually test for [].

diff --git a/scripts/gui/uimenu.m b/scripts/gui/uimenu.m
--- a/scripts/gui/uimenu.m
+++ b/scripts/gui/uimenu.m
@@ -114,27 +114,25 @@ endfunction
 %!   assert (get (ui, "separator"), "off");
 %!   assert (get (ui, "enable"), "on");
 %!   assert (get (ui, "position"), 0);
 %! unwind_protect_cleanup
 %!   close (hf);
 %!   graphics_toolkit (toolkit);
 %! end_unwind_protect
 
-## check for top level menus file, edit, and help
+## check for top level menus file and edit
 %!testif HAVE_FLTK
 %! toolkit = graphics_toolkit ("fltk");
 %! hf = figure ("visible", "off");
 %! unwind_protect
 %!   uif = findall (hf, "label", "&file");
 %!   assert (ishghandle (uif));
 %!   uie = findall (hf, "label", "&edit");
 %!   assert (ishghandle (uie));
-%!   uih = findall (hf, "label", "&help");
-%!   assert (ishghandle (uih));
 %! unwind_protect_cleanup
 %!   close (hf);
 %!   graphics_toolkit (toolkit);
 %! end_unwind_protect
 
 %!testif HAVE_FLTK
 %! toolkit = graphics_toolkit ("fltk");
 %! hf = figure ("visible", "off");
diff --git a/scripts/linear-algebra/isbanded.m b/scripts/linear-algebra/isbanded.m
--- a/scripts/linear-algebra/isbanded.m
+++ b/scripts/linear-algebra/isbanded.m
@@ -53,17 +53,17 @@ function retval = isbanded (A, lower, up
     endif
   endif
 
 endfunction
 
 
 %!assert (! isbanded ("string", 0, 0))
 %!assert (! isbanded (zeros (2,2,2), 0, 0))
-%!assert (isbanded ([], 0, 0))
+%!assert (isbanded ([], 0, 0), [])
 %!assert (isbanded (1,0,0))
 %!assert (isbanded (1,10,10))
 
 %!assert (isbanded ([1, 1],1,1))
 %!assert (isbanded ([1; 1],1,1))
 %!assert (isbanded (eye(10),0,0))
 %!assert (isbanded (eye(10),1,1))
 %!assert (isbanded (i*eye(10),1,1))
